topo/dva/behaviors_block-yellow.yaml

```diff
@@ -5,6 +5,3 @@ device_mac_behaviors:
   "9a:02:57:1e:8f:02":
     segment: BOS
     role: green
-  "9a:02:57:1e:8f:03":
-    segment: BOS
-    role: yellow
```

Comment:
I think we can modify the file name so that it makes more sense... I named it "block-yellow" because devices assigned with role "yellow" in this file will be blocked according to the ACL for that role. Since the "yellow" devices are removed, I think it is better to give the file a new name to reflect the content/test case.

Reply:
As per other comment -- anything not required with radius integration change should be done in a separate PR.

Reply:
The thing is the behaviors file is the static way of conveying behavior to faucetizer. This overwrites any result sent from the authenticator. This change would remove static behavior for faux 3 that we are testing.

Reply:
After reading both your comments more carefully I agree that the filename should be changed -- sorry, I didn't quite understand the meaning before. Also the devices_block-yellow.yaml file seems misnamed too (it actually has nothing to do with blocking yellow, that's just when it's used).